src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import './App.css';
+import Navbar from './components/layout';
+import Categories from './components/categories';
+import Books from './components/showbooks';
+
+function App() {
+  return (
+    <div className="wraper">
+      <Routes>
+        <Route element={<Navbar />}>
+          <Route index element={<Books />} />
+          <Route path="Categories" element={<Categories />} />
```

Comment:
- [ ] Please 🙏🏽 correct your code for the better practice of React components, Navbar is not a route, all the route has path and elements attributes, all endpoints should be write in lowercase, e.g. 👇🏽 

```javascript
<div className="wraper">
<Navbar />
      <Routes>
          <Route path="/" element={<Books />} />
          <Route path="/categories" element={<Categories />} />
```